listenbrainz/webserver/static/js/src/APIService.ts

```diff
@@ -420,7 +420,7 @@ export default class APIService {
   getUserArtistMap = async (
     userName: string,
     range: UserStatsAPIRange = "all_time",
-    forceRecalculate: boolean = false
+    forceRecalculate: boolean = true
```

Comment:
Revert this before merging but invalidate all existing artist map stats manually from db.